classrooms/forms.py

```
@@ -0,0 +1,29 @@
+    class Meta:
+        model = Classroom
+        fields = ['title', 'short_introduction', 'course_description', 
+                  'course_image', 'preview_video_link',
+                  'upcoming', 'paid_course',
+                  'course_price']
+
+    def __init__(self, *args, **kwargs):
+        self.user = kwargs.pop('user', None)
+        super(ClassroomForm, self).__init__(*args, **kwargs)
+
+    def save(self, commit=True):
```

Ruff (0.8.2)

Missing return type annotation for public function `save`

(ANN201)